tasks/config.yml

```diff
@@ -0,0 +1,47 @@
+- name: Set openvpn service name - systemd
+  set_fact:
+    openvpn_service_name: "openvpn@{{openvpn_config_file}}.service"
+  when: ansible_service_mgr == "systemd" or (docker_stat_result.stat is defined and docker_stat_result.stat.exists == True)
+
+- name: Create openvpn config file
+  template:
+    src: server.conf.j2
+    dest: "{{ openvpn_base_dir }}/{{ openvpn_config_file }}.conf"
+    owner: root
+    group: root
+  notify:
+    - Restart openvpn
+
+- name: Install LDAP config
+  template:
+    src: ldap.conf.j2
+    dest: "{{ openvpn_base_dir }}/auth/ldap.conf"
+    owner: root
+    group: root
+  when: openvpn_use_ldap
+
+- name: Copy openvpn logrotate config file
+  copy:
+    src: openvpn_logrotate.conf
+    dest: /etc/logrotate.d/openvpn.conf
+    owner: root
+    group: root
+    mode: 0400
```

Comment:
You could give `644`, it's not secret and making it readable helps troubleshooting

Reply:
fixed